templates/case-studies/example.html

```diff
@@ -1,72 +1,85 @@
+      <p>To meet this goal, ESA engaged Canonical to deliver the architecture, design and deployment of Kubernetes and Ceph for persistent storage. ESA also chose Canonical’s Ceph distribution to consolidate the various storage technologies in use. Canonical’s Kubernetes distribution allowed ESA’s Missions Operations Infrastructure – IT (MOI-IT) team to easily deploy its workloads, integrated with the existing IT infrastructure services, giving it a greater ability to support ESA missions and those of its partners, with automatic updates and deployment, 24/7 monitoring and reduced system complexity.</p>
+      <p>Kubernetes was perfect for ESA’s next generation multi-mission platform at ESOC. This open source system helps organisations orchestrate their workloads more efficiently with automation. However, Kubernetes’ fast-moving and steep learning curve makes it more demanding to manage. To reduce the engineering workload and management requirements of this new system, ESA chose Canonical to manage its Kubernetes deployments, providing updates, monitoring and maintenance.</p>
+      <p>The systems Canonical deployed and now manage for ESA mean that ESOC engineering teams can quickly and centrally manage and deploy mission-critical infrastructure and software, with more security, reliability, and control, and with less downtime and reduced risk of system failures.</p>
+    {%- endif -%}
+  {%- endcall -%}
 
   <!-- Quote section-->
-  {% import "case-studies/templates/_macros-quote.jinja" as quote_template %}
-  {{ quote_template.quote(
-      quote="Enterprises need modular, cloud-native application platforms that accelerate how they build, run, and manage their applications without compromising on their compliance, security, or support requirements.",
-      name="Justin Boitano", 
-      job_title="Vice President and General Manager of Customer and Ecosystem Enabling Division, Intel's Network and Edge Solutions Group"
-    )
-  }}
+  {%- call(slot) quote_wrapper(
+    quote_heading_level=4,
+    quote_text="Enterprises need modular, cloud-native application platforms that accelerate how they build, run, and manage their applications without compromising on their compliance, security, or support requirements.",
+    citation_source_name="Justin Boitano",
+    citation_source_title="Vice President and General Manager of Customer and Ecosystem Enabling Division, Intel's Network and Edge Solutions Group"
+    ) 
+  -%}
+  {%- endcall -%}
 
   <!-- Image section-->
-  {% from "case-studies/templates/_macros-image.jinja" import image_span %}
-  {%- call(slot) image_span(caption="Copyright: ESA/DLR/FU Berlin/G. Michael CC BY-SA 3.0 IGO") -%}
+  {%- call(slot) image_wrapper(caption="Copyright: ESA/DLR/FU Berlin/G. Michael CC BY-SA 3.0 IGO") -%}
     {%- if slot == 'image_content' %}
       {{ image(url="https://assets.ubuntu.com/v1/c3f56868-Global_Mars_in_colour 1.png",
-                alt="Mars in colour",
+                alt="",
```

Comment:
Why remove the alt text?

Reply:
Thanks for the catch, this must've been left out when rebasing.